www/js/metrics/CarbonTextCard.tsx

```diff
@@ -55,7 +56,7 @@ const CarbonTextCard = ({ userMetrics, aggMetrics }: Props) => {
         };
         const label = `${t('main-metrics.prev-week')} (${formatDateRangeOfDays(lastWeekDistance)})`;
         if (userPrevWeek.low == userPrevWeek.high)
-          textList.push({ label: label, value: Math.round(userPrevWeek.low) });
+          textList.push({ label: label, value: `${Math.round(userPrevWeek.low)}` });
```

Comment:
I assume that this is part of "cleanup formatting". But can you clarify why formatting this as text right here is better?

Reply:
I think it's because `Math.round` returns a `number` and we want a `string`. So this was just a quick way to cast to a string.
We could alternatively do `"" + x` or `String(x)` or `x.toString()`